revista-ladinia.csl

```diff
@@ -0,0 +1,163 @@
+    <updated>2022-04-22T12:16:36+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="it">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="no-publisher">N.N.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="de">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="no-publisher">N.N.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="no-publisher">N.N.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" et-al-use-first="1">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <et-al term="and others"/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name name-as-sort-order="all" delimiter="/">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <label prefix=" (" suffix=")"/>
+          <et-al term="and others"/>
+        </names>
+      </else-if>
+      <else>
+        <text term="no-publisher"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author-bibliography">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name name-as-sort-order="all" delimiter="/">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <et-al/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor translator">
+          <name name-as-sort-order="all" delimiter="/">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <label prefix=" (" suffix=")"/>
+          <et-al/>
+        </names>
+      </else-if>
+      <else>
+        <text term="no-publisher"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued-year">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher-place">
+    <choose>
```

Comment:
I think you'll want to restrict this macro to some item types -- e.g. you don't want s.l. for journal articles.